src/webgpu/util/conversion.ts

```diff
@@ -391,21 +392,24 @@ export function f32Bits(bits: number): Scalar {
 }
 /** Create an f16 from a bit representation, a uint16 represented as a JS `number`. */
 export function f16Bits(bits: number): Scalar {
-  const arr = new Uint32Array([bits]);
+  const arr = new Uint16Array([bits]);
   return new Scalar(TypeF16, float16BitsToFloat32(bits), arr);
```

Comment:
Should the arg to float16BitsToFloat32 be arr[0] instead of bits?

Reply:
oh,  float16BitsToFloat32 only pays attention to the bottom 16 bits of its argument.  never mind.  :-)